delivery.info.yml

```diff
@@ -1,11 +1,12 @@
 name: 'Delivery'
 type: module
 description: 'Deliver content changes between workspaces.'
-core: 8.x
```

Comment:
For this release, removing `core` means we drop support for everything before 8.8 - is that intended / fine? I'm not sure if that will cause difficulties for upgrading (e.g. if someone wanted to upgrade this module to this version and then upgrade core to 8.8).

Given that it's mostly used internally at Amazee, we. can perhaps just align with ALGM on whether they are fine with this? @fjgarlin

Reply:
Yeah, this is a bit of a pain, but I don't see how we can support both on a technical level because the APIs have started to diverge now.

Reply:
Ok to require 8.8 as specially this is used on newer projects